modules/websocket/wsl_server.cpp

```diff
@@ -165,7 +165,7 @@ Error WSLServer::listen(int p_port, const Vector<String> p_protocols, bool gd_mp
 	for (int i = 0; i < p_protocols.size(); i++) {
 		pw[i] = p_protocols[i].strip_edges();
 	}
-	_server->listen(p_port);
+	_server->listen(p_port, IP_Address(bind_ip));
```

Comment:
`IP_Address(bind_ip)` will be redundant when `bind_ip` is changed to `IP_Address` type

Reply:
should be fixed now